apps/ios/Shared/Views/Chat/ChatItem/FramedItemView.swift

```diff
@@ -180,7 +186,7 @@ struct FramedItemView: View {
         .foregroundColor(.secondary)
         .padding(.horizontal, 12)
         .padding(.top, 6)
-        .padding(.bottom, chatItem.quotedItem == nil ? 6 : 0) // TODO think how to regroup
+        .padding(.bottom, padBottom ? 6 : 0)
```

Comment:
ah, so it was simply determined inside... why pass from outside?

Reply:
to avoid extra padding between marked deleted and forwarded headers

Reply:
why not amend the condition locally?

Reply:
it would affect forwarded items themselves them - same view is used for them